qt_binder/tests/test_widgets.py

```diff
@@ -14,13 +14,14 @@
 
 import unittest
 
-from traits.api import Undefined, pop_exception_handler, push_exception_handler
 from traits.testing.unittest_tools import UnittestTools
 
+from .test_raw_widgets import _BaseTestWithGui
```

Comment:
If we're going to use this in multiple places, maybe put it on its own somewhere?